api/v1beta1/loadbalancer_types.go

```diff
@@ -104,6 +108,16 @@ type LoadBalancerOptions struct {
 	// If empty it is enabled for all ports. Only has an affect if TCPProxyProtocol is enabled.
 	// +optional
 	TCPProxyProtocolPortsFilter []int32 `json:"tcpProxyProtocolPortFilter,omitempty"`
+	// TCPIdleTimeout sets TCP idle Timeout for all TCP connections from this LoadBalancer.
+	// Value is in Seconds. With 0 you disable the idle timeout, be careful this can lead to side effects.
+	// Default is 3600s (1 hour).
+	// +optional
+	TCPIdleTimeout *int `json:"tcpIdleTimeout,omitempty"`
```

Comment:
I wonder why you don't support durations here? [k8s](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration) supports them, [Go](https://pkg.go.dev/time#Duration) supports them, [protobuf](https://pkg.go.dev/google.golang.org/protobuf/types/known/durationpb) supports them too.

Durations make it clearer what unit are your talking about without looking at the documentation and allow you to go sub-second when your timings get tighter.

Reply:
Good point, changed in #73